spex/extensions/spex-detach/scripts/spex-detach.py

```
@@ -0,0 +1,493 @@
+        dest_brainstorm = os.path.join(archive_dir, "brainstorm")
+        shutil.copytree("brainstorm", dest_brainstorm, dirs_exist_ok=True, symlinks=True)
+        files_copied += sum(len(files) for _, _, files in os.walk("brainstorm"))
+        if move:
+            source_dirs_to_delete.append("brainstorm")
+
+    committed = False
+    should_commit = auto_commit or read_config(".archive.auto_commit", "true") == "true"
+    if should_commit and git_ok("rev-parse", "--git-dir", cwd=target):
+        git("add", os.path.join(project, feature), cwd=target)
+        if not git_ok("diff", "--cached", "--quiet", cwd=target):
+            r = subprocess.run(
```

Ruff (0.15.21)

[error] `subprocess` call: check for execution of untrusted input

(S603)